releasenotes/notes/fix-raw-feature-vector-copy-20534d595a0056de.yaml

```
@@ -0,0 +1,5 @@
+---
+fixes:
+  - |
+    After this ``RawFeatureVector`` fix, ``copy()`` and ``bind_parameters()`` methods work correctly.
```

I think if users read the release notes they should understand what the issue was we fixed, so we could be a bit more detailed in the descriptions 🙂 
```suggestion
    Fix a bug, where :meth:`qiskit_machine_learning.circuit.library.RawFeatureVector.copy`
    didn't copy all internal settings which could lead to issues with the copied circuit.
    As a consequence :meth:`qiskit_machine_learning.circuit.library.RawFeatureVector.bind_parameters`
    is also fixed.
    didn't  ``RawFeatureVector`` fix, ``copy()`` and ``bind_parameters()`` methods work correctly.
```

---

Updated as you suggested.